src/console/controllers/ProjectConfigController.php

```diff
@@ -35,6 +37,23 @@ class ProjectConfigController extends Controller
      */
     public bool $invert = false;
 
+    /**
+     * @var bool Whether to get values from the loaded config, or directly from file.
+     */
+    public bool $fromFile = false;
```

Comment:
This should probably be `external` to match wording from `craft\services\ProjectConfig::get()`’s `$getFromExternalConfig` argument.

Reply:
Also needs a `@since 4.1.0`

Reply:
Yeah, that was my initial though too…I went with `fromFile` because I didn't think `external` would really make sense to anyone not intimately familiar with the PC service…but alas, changed it.